modules/tensor_mechanics/src/bcs/Pressure.C

```diff
@@ -19,6 +21,8 @@ Pressure::validParams()
   InputParameters params = IntegratedBC::validParams();
   params.addClassDescription("Applies a pressure on a given boundary in a given direction");
   params.addRequiredParam<unsigned int>("component", "The component for the pressure");
+  params.addRequiredCoupledVar("displacements",
+                               "The string of displacements suitable for the problem statement");
```

Comment:
The now required `displacements` parameter will require some app fixes. Also `component` is somewhat redundant, as it can be inferred from the variable the BC is acting on and the `displacements` vector.

Reply:
I've adjusted how components are handled.  I also put in a Marmot MR.  Bison will require coordination and a submodule update.